sources/packages/web/src/components/institutions/locations/LocationEditForm.vue

```diff
@@ -19,14 +28,17 @@ export default {
   },
   emits: ["updateInstitutionLocation"],
   setup(_props: any, context: SetupContext) {
-    const updateInstitutionLocation = async (
-      data: InstitutionLocationFormAPIOutDTO,
-    ) => {
+    const processing = ref(false);
+    const updateInstitutionLocation = async (form: FormIOForm) => {
```

Comment:
We can define the type of form data by using FormIOForm<T>

![image](https://user-images.githubusercontent.com/54600590/186250983-8ba26d80-4412-44c0-bc66-aa2c469f386b.png)

Reply:
this is not part of my ticket https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/1084

Reply:
As this is the refactor done to use the common component formio-container, I would still consider it as part of the ticket to use the explicit data type for  FormIOForm.

Reply:
okay, you can add it to other UI tickets @dheepak-aot